### PR TITLE
Docker and Helm

### PR DESCRIPTION
**Pull request by awolfe76**
Adds necessary files for v2 deployment.

### CODE REVIEW

File: kubernetes/hmda-platform-ui/templates/service.yaml [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+metadata:
+  labels: 
+    service: {{ include "hmda-platform-ui.fullname" . }}-ambassador
+  name: {{ include "hmda-platform-ui.fullname" . }}-ambassador
+  annotations:
+    getambassador.io/config: |
+      apiVersion: ambassador/v0
+      kind: Mapping
+      name: tools_mapping
+      prefix: /tools/
```
Comment (wpears) [inline kubernetes/hmda-platform-ui/templates/service.yaml]: This should probably be `/filing/` or something similar

**Review by wpears — CHANGES_REQUESTED** [review]

File: kubernetes/hmda-platform-ui/templates/service.yaml (hunk 1 above)
Comment (awolfe76) [reply kubernetes/hmda-platform-ui/templates/service.yaml]: Good catch. I thought I got them all. 😛 

e37b451

File: kubernetes/hmda-platform-ui/templates/service.yaml [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+kind: Service
+metadata:
+  labels: 
+    service: {{ include "hmda-platform-ui.fullname" . }}-ambassador
+  name: {{ include "hmda-platform-ui.fullname" . }}-ambassador
+  annotations:
+    getambassador.io/config: |
+      apiVersion: ambassador/v0
+      kind: Mapping
+      name: tools_mapping
```
Comment (kgudel) [inline kubernetes/hmda-platform-ui/templates/service.yaml]: `tools_mapping` is already in use. This should be changed to something like `platform_ui_mapping`.

**Review by kgudel — CHANGES_REQUESTED** [review]
Other than one small issue this all builds properly in jenkins and deploys correctly to kubernetes.

File: kubernetes/hmda-platform-ui/templates/service.yaml (hunk 2 above)
Comment (awolfe76) [reply kubernetes/hmda-platform-ui/templates/service.yaml]: 65ea204

**Review by wpears — APPROVED** [review]

**Pull request merged**